docs/source/tune_cli.rst

```diff
@@ -39,19 +39,25 @@ to download files using the CLI.
 Download a model
 ----------------
 
-The ``tune download <path>`` command downloads any model from the Hugging Face Hub.
+The ``tune download <path>`` command downloads any model from the Hugging Face or Kaggle Model Hub.
 
 .. list-table::
    :widths: 30 60
 
    * - \--output-dir
-     - Directory in which to save the model.
+     - Directory in which to save the model. Note: this is option not yet supported when `--source` is set to `kaggle`.
    * - \--output-dir-use-symlinks
      - To be used with `output-dir`. If set to 'auto', the cache directory will be used and the file will be either duplicated or symlinked to the local directory depending on its size. It set to `True`, a symlink will be created, no matter the file size. If set to `False`, the file will either be duplicated from cache (if already exists) or downloaded from the Hub and not cached.
    * - \--hf-token
      - Hugging Face API token. Needed for gated models like Llama.
    * - \--ignore-patterns
```

Comment:
just noticed that `--ignore-patterns` should probably emit a warning too for Kaggle models. It has a default value though (`"*.safetensors"`), which should probably be kept for backwards compatibility. 

https://github.com/pytorch/torchtune/blob/1eb7785321dcc651e5bf5b44842f790b7b873aa5/torchtune/_cli/download.py#L90-L97

So we'd need a way to distinguish when a user explicitly sets it 🤔

Reply:
Yep. For example, our Transformers variation of Gemma mirrors the files on the Hugging Face page:

https://www.kaggle.com/models/google/gemma/transformers

Reply:
Ah I see. Yeah we should log a warning about this too for Kaggle and honestly, we should just get rid of this default altogether IMO. When we were bootstrapping up, we supported only Meta model formats to make things easier and so didn't want to download double weights, but now the majority of our models are safetensor format.

Reply:
Addressed here: f91607bf813153d16c0d4f308c70b32fbb4e2bd9